src/main/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -14,7 +14,10 @@ import { MomentFactory } from 'shared/momentFactory'
 
 export class ClaimantResponseConverter {
 
-  public static convertToClaimantResponse (draftClaimantResponse: DraftClaimantResponse): ClaimantResponse {
```

Comment:
@robertparkinson @varma02 @rattur pls review - these changes are to fix claimant response with refer to judge formalisation option (2 of 2)